db/liquibase/changesets/20180917_entity.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="avram" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="avram" author="ansingh" id="entity6">
+        <createTable tableName="ENTITY">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="name" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="collection" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="entityBody" type="JSONB">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addUniqueConstraint columnNames="name, collection" constraintName="IDX_ENTITY_COLLECTION_UNIQUE" tableName="ENTITY"/>
+        <addForeignKeyConstraint baseTableName="ENTITY" baseColumnNames="collection" constraintName="FK_COLLECTION"
```

Comment:
Same as above about also adding indexes on the fields this table will be queried by.